### ISSUE TITLE
Allow DisableAttribute to be multiples

### ISSUE THREAD

**Issue by drdamour** [Azure/azure-webjobs-sdk#2626]
I'd like to annotate Functions with multiple disables so i could have
```
[Disable("DISABLE_EXPERIMENTAL")]
[Disable("DISABLE_BROKEN")]
[FunctionName("ExperimentalBrokenStuff1")]
public static int BrokenExperimentalStuff1()
```

and it'd be disabled if either DISABLE_EXPERIMENTAL or DISABLE_BROKEN settings resolve true

**Comment by fabiocav** [comment]
@drdamour can you share more details of your use case? Thanks!

**Comment by drdamour** [comment]
@fabiocav what details are u looking for?

**Comment by fabiocav** [comment]
@drdamour just trying to understand your scenario and why what we currently have is a blocker. This would help us evaluate and prioritize the feature on our side.

**Comment by drdamour** [comment]
@fabiocav yeah but i laid out my scenario when i opened the ticket..can you please explain the additional detail you're looking for?

**Comment by fabiocav** [comment]
@drdamour what is described above is what and how you'd like to do it. Can you please elaborate on the why? Why is what we have now insufficient for what you're trying to accomplish? 

Thanks!

**Comment by drdamour** [comment]
Ok.  Because i want one code base in all my environments.  In prod i want to disable the broken and experimental, in test i want to only disable broken, and in dev i want all enabled (disable none).  Is that what u were looking for?

**Comment by v-anvari** [comment]
Hi @drdamour , Can you give us the piece of code that you think is busted. We also require the repro steps for us to understand the scenario better. It would be helpful if you could provide us any related links that you are referring to.

**Comment by drdamour** [comment]
@v-anvari nothing is busted, this is an enhancement request. I provided the code in the initial post

**Comment by v-anvari** [comment]
Adding this as an enhancement request 

Cc: @fabiocav , Can we consider this feature to be included in later releases?

**Comment by v-anvari** [comment]
Hi @drdamour , you can also raise a new feature request or post a new idea in [feedback.azure.com-azure-web-apps](https://feedback.azure.com/forums/169385-web-apps)
